lib/thermal.go

```diff
@@ -53,7 +53,11 @@ func (t ThermalPlugin) FetchMetrics() (map[string]float64, error) {
 
 		temperature, err := ioutil.ReadFile(filepath.Join(file, "temp"))
 		if err != nil {
-			return nil, err
+			if strings.Contains(err.Error(), "no data available") {
+				continue
+			} else {
+				return nil, err
+			}
```

Comment:
In my environment (Ubuntu 22.04.4 on Minisforum UN100D), the following error was occurred.

```
$ cat /sys/class/thermal/thermal_zone*/type
acpitz
iwlwifi_1
x86_pkg_temp

$ cat /sys/class/thermal/thermal_zone*/temp'
27800
cat: /sys/class/thermal/thermal_zone1/temp: No data available
49000
```